crypto/src/conversation.rs

```diff
@@ -341,6 +350,39 @@ impl MlsConversation {
         Ok(message)
     }
 
+    fn get_self_index(&self, backend: &MlsCryptoProvider) -> CryptoResult<usize> {
```

Comment:
* IMO `get_self_tree_index` would be a better name
* Add a small documentation
* There are no tests for this method